filebeat/module/kibana/log/config/log.yml

```diff
@@ -5,10 +5,20 @@ paths:
 {{ end }}
 exclude_files: [".gz$"]
 
-json.keys_under_root: false
```

Comment:
When this is set to `true`, Filebat overwrites all fields correctly, but it also replaces the log entry @timestamp with Filebeat's. And that would make the ingested data inconsistent